Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3882,7 +3882,7 @@ MinimumZoomFontSize:
 
 MockCaptureDevicesEnabled:
   type: bool
-  status: internal
+  status: embedder
```

Comment:
Having it internal allows to toggle it on/off in MiniBrowser as well as Safari outside of WebRTC develop menu.
Can we keep it as Internal?